.github/workflows/kind.yml

```diff
@@ -86,7 +86,6 @@ jobs:
     - name: Test Coverage and Report Generation
       run: |
         make test-coverage | tee report_unit.json
-        make coverage-verify
```

Comment:
Why was this removed?

Reply:
Incompatibility with Go 1.22

Reply:
Details: https://github.com/wadey/gocovmerge doesn't have a `go.mod`, so it's pulling the latest of everything, so we can't use Go 1.21 to run it.